AUTHORS

```diff
@@ -512,59 +528,156 @@ Cam Swords <cam.swords@gmail.com>
 Paul Loyd <pavelko95@gmail.com>
 Benjamin Waters <benjamin.waters@outlook.com>
 Lev Gimelfarb <lev.gimelfarb@gmail.com>
-Peter Flannery <flannery.peter@ntlworld.com>
+Yorkie <l900422@vip.qq.com>
+pflannery <pflannery@users.noreply.github.com>
 Tuğrul Topuz <tugrultopuz@gmail.com>
 Lorenz Leutgeb <lorenz.leutgeb@gmail.com>
-Brandon Cheng <bcheng.gt@gmail.com>
-Alexis Campailla <alexis@janeasystems.com>
-Yang Tianyang <contact@ayanamist.com>
+ayanamist <contact@ayanamist.com>
+gluxon <bcheng.gt@gmail.com>
 Tom Gallacher <tomgallacher23@gmail.com>
+svenpanne@chromium.org <svenpanne@chromium.org>
 Jo Liss <joliss42@gmail.com>
 Jun Ma <roammm@gmail.com>
 Jacob Hoffman-Andrews <github@hoffman-andrews.com>
-Keith M Wesolowski <wesolows@joyent.com>
+Keith M Wesolowski <wesolows@foobazco.org>
 Maxime Quandalle <maxime.quandalle@gmail.com>
-Doron Pagot <doronpagot@gmail.com>
 Oguz Bastemur <obastemur@gmail.com>
+Yuriy Nemtsov <nemtsov@gmail.com>
+Benjamin Waters <ben25890@gmail.com>
+iamdoron <doronpagot@gmail.com>
+Austin Moran <moraustin@gmail.com>
 Kenan Sulayman <kenan@sly.mn>
-Christian Schulz <me@rndm.de>
+Christian <me@rndm.de>
+orangemocha@github.com <orangemocha@github.com>
```

Comment:
Another .mailmap entry: @orangemocha is Alexis Campailla.  The one above should probably go into the .mailmap as well.